guides/ecto.md

```diff
@@ -11,7 +11,7 @@ Newly generated Phoenix projects include Ecto with the PostgreSQL adapter by def
 
 For a thorough, general guide for Ecto, check out the [Ecto getting started guide](https://hexdocs.pm/ecto/getting-started.html). For an overview of all Ecto specific mix tasks for Phoenix, see the [mix tasks guide](phoenix_mix_tasks.html#ecto-specific-mix-tasks).
 
-This guide assumes that we have generated our new application with Ecto integration and that we will be using PostgreSQL. For instructions on switching to MySQL, please see the [Using MySQL](ecto.html#using-mysql) section.
+This guide assumes that we have generated our new application with Ecto integration and that we will be using PostgreSQL. For instructions on switching to MySQL, please see the [Using MySQL](ecto.md#using-mysql) section.
```

Comment:
it needs to stay as .html because otherwise it will 404 when rendered: https://hexdocs.pm/phoenix/ecto.html#using-mysql

Reply:
Huh interesting. I'll go ahead and close this PR then.

Thanks for the quick response, & sorry for the PR bloat.

Reply:
Oh, I just realised that because we're already on this page if we change it like this:

```markdown
[Using MySQL](#using-mysql)
```

it will work in both cases :D

Reply:
Oh! In that case, I'll go ahead and re-open and make that change!

Reply:
If you would prefer this to be one commit instead of two I'll be able to squash in about an hour when I'm at computer configured for my H12 GitHub account (I'm currently on a non-personal machine, and limited to making changes through the browser >.<).

Reply:
maintainers have an option to squash when merging so no need to worry about it!